helmfile/values/podsecuritypolicies/common/fluentd.yaml.gotmpl

```diff
@@ -6,9 +6,7 @@ constraints:
         app.kubernetes.io/instance: fluentd-forwarder
       allow:
         allowPrivilegeEscalation: true
-        allowedCapabilities:
-          - AUDIT_READ
-          - SYSLOG
+        allowedCapabilities: []
```

Comment:
Are these not needed anymore?

Reply:
They aren't set in the securityContext so its not possible to validate, thus also why the capabilities mutation has been disabled.

Reply:
Ah okay